cornedbeef/build.gradle

```diff
@@ -7,13 +7,12 @@ apply plugin: 'com.android.library'
 
 android {
     compileSdkVersion 23
```

Comment:
This is cheeky (and optional :)) but since we're updating Gradle which is unrelated to the change anyway and we're making a major version bump, we could update the compileSdkVersion, targetSdkVersion and support library.